sanic/config.py

```diff
@@ -83,6 +83,8 @@ def from_pyfile(self, filename):
         except IOError as e:
             e.strerror = 'Unable to load configuration file (%s)' % e.strerror
             raise
+        except Exception:
+            raise SyntaxError('Invalid syntax in config file.')
```

Comment:
I'm a bit confused here because when I `compile('VALUE = some value', 'name.py', 'exec')` I actually get a `SyntaxError`, which means the original implementation should raise a `Syntaxerror`, what were you seeing?

Reply:
If you put `VALUE = 1/0` in the config file you get a `ZeroDivisionError`. I put `except Exception` because I want to catch all type of exceptions.

Reply:
I don't think we should wrap all exceptions up, and raise `Syntaxerror` instead. It also hides the true error. If it raises `ZeroDivisionError` because of `VALUE = 1/0`, it should just raise `ZeroDivisionError`.

Reply:
@yunstanford I changed it and added an extra test